README.md

```diff
@@ -1,18 +1,76 @@
 # help-center-content
 
-Repository assembling VTEX's Help Center content
+Welcome to the [VTEX Help Center Portal](https://help.vtex.com/) content repository! Here you will find the archives for all articles included in this documentation portal. It is managed by the [Tech Writing team](https://github.com/vtexdocs/help-center-content/graphs/contributors) at VTEX, so if you have any questions about the repository, do not hesitate to contact any of them.
 
 ## In this repository
 
 You will find the following folders in this repository:
 
--TBD
+- **.github/workflows**: store workflows needed to migrate content from Contentful to this repository.
+- **docs-utils**: store scripts used to perform adjustments to documentation imported from Contentful.
+- **docs**: stores the markdown files of all our docs. Contains subfolders with names of the three locales (PT, EN, and ES), and within each of them, the following categories:
 
-## Contributing with Help Center Portal documentation
+  - **announcements**: Announcement markdown files.
+  - **faq**: FAQ markdown files.
+  - **know-issues**: Know-issues markdown files.
+  - **tracks**: Tracks markdown files.
+  - **troubleshooting**: Troubleshooting markdown files.
+  - **tutorials**: Tutorials markdown files.
+- **readme-images**: store images used in the Readme's document.
 
-We're so glad you're here! Thanks for being interested.
+## Management of Help Center Portal documentation
 
-### What is the maximum image size I can use in an article?
+In this document we will address the following topics related to the Help Center Portal:
+
+- [Publication of new articles](#publishing-a-new-article)
+- [Update navigation sidebar](#updating-navigation-sidebar)
+- [Update of published articles](#updating-a-published-article)
+- [Create a redirect](#creating-a-redirect)
+- [Settings for specific content](#settings-for-specific-content)
+
+
+
+
+### Publishing a new article
+
+Follow the steps below to create a new article in the Help Center Portal:
+
+1. Open a branch in the [help-portal-content](https://github.com/vtexdocs/help-center-content) repository.
+2. Add a new file in the [desired folder](#in-this-repository), and write the content of the article according to the [Template --> TBD](https://github.com/vtexdocs/dev-portal-content/blob/main/templates/guide_template.md).
+    > ⚠️ Make sure the file name (slug) is unique and an exact copy of the article title.
+
+3. [Add images](#adding-an-image-to-the-article) to the article, if necessary.
+3. Fill in the [front matter](#filling-in-front-matter-information) of the article.
+4. Submit your PR for review on GitHub.
+5. Send the PR link in the `#helpcenter-portal-pr` Slack channel for review by another technical writer.
```

Comment:
```suggestion
5. Send the PR link in the `#helpcenter-pr` Slack channel for review by another technical writer.
```

Reply:
are we creating a new channel? or renaming #dev-portal-pr to something like #documentation-pr? @PedroAntunesCosta